app/components/footerlinks/footerlinks.less

```diff
@@ -0,0 +1,72 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2017, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+.footer-link {
+  display: inline-block;
+  font-size: 14px;
+
+  &.social-media {
+    display: flex;
+  }
+
+  &.large-format-only {
+    @media(max-width: @screen-md-max) {
+      display: none;
+    }
+  }
+
+  a {
+    &:extend(.link-secondary all);
+
+    svg {
+      fill: @gray-dark;
+      &:hover {
+        fill: @blue-green-light;
+      }
+    }
+  }
+
+  img {
+    margin: 0 12px;
+  }
+}
+
+@imgHeight: 32px;
+
+.footer-twitter {
+  width: 44.8px;
```

Comment:
Math 🙂  I was scaling exactly to match the size of the Facebook logo from the original dimensions (slightly larger) of the Twitter logo.